### PR TITLE
docs: fix License url

### PR DESCRIPTION
**Pull request by popomore**
##### Checklist

- [ ] `npm test` passes
- [ ] tests and/or benchmarks are included
- [ ] documentation is changed or added
- [ ] commit message follows commit guidelines

### CODE REVIEW

**Comment by popomore** [comment]
https://github.com/eggjs/egg/pull/989

File: README.md [hunk 1]
```diff
@@ -269,15 +269,16 @@ exports.view = {
...
+see [config/config.default.js](https://github.com/eggjs/egg-view/blob/master/config/config.default.js) for more detail.
 
 ## Questions & Suggestions
 
 Please open an issue [here](https://github.com/eggjs/egg/issues).
 
 ## License
 
-[MIT](LICENSE)
+[MIT](https://github.com/eggjs/egg-view/blob/master/LICENSE)
```
Comment (fengmk2) [inline README.md]: 这种写法不行了？

**Comment by fengmk2** [comment]
看到issue了，那以后都得写完整url了

**Comment by popomore** [comment]
+ egg-view@1.1.1

**Pull request merged**